smol-flashing-firmware.md: Add note for DFU mode via Serial, Link to pre-compiled firmware (#378)
SlimeVR/SlimeVR-Docs-Site: src/diy/smol-slimes/smol-flashing-firmware.md

### Flashing using nRF Connect for Desktop
1. Open "Programmer" in the nRF Connect.
1. Put the device into DFU mode using either of these methods:
    1. Press the reset button — the LED should begin to fade on and off, indicating the device is in DFU Mode. For the eByte dongle, this is the right button. For the Nordic dongle, it is the side button (not the round white button).

>     2. If the dongle already has existing SlimeNRF firmware, using the Serial Terminal in nRF Connect, and enter ```dfu``` to put the device into DFU mode.

Author: This wasn't grammaring anymore
```suggestion
    2. If the dongle already has existing SlimeNRF firmware, use the Serial Terminal in nRF Connect, and enter ```dfu``` to put the device into DFU mode.
```
